roboteam_ai/test/ControlTests/ControlUtilsTest.cpp

```diff
@@ -211,4 +212,18 @@ TEST(ControlUtils, it_calculates_forces) {
     force = cr::ControlUtils::calculateForce(Vector2(0, 1), - 2, 3);
     EXPECT_DOUBLE_EQ(force.x, 0);
     EXPECT_DOUBLE_EQ(force.y, - 2);
+}
+
+TEST(ControlUtils, getGenevaAimTest) {
```

Comment:
`ASSERT_GT(0.0001, ...` really? this does not test anything

Reply:
Done